src/System.Threading.Thread/src/System/Threading/Thread.cs

```diff
@@ -16,11 +16,14 @@ public sealed partial class Thread : CriticalFinalizerObject
         [ThreadStatic]
         private static Thread t_currentThread;
 
+        [ThreadStatic]
+        private static IPrincipal s_principal;
```

Comment:
This should have `t_` prefix. Also, I would put the AsyncLocal and this static next to each other since they logically belong together.

Reply:
Right: https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md

Reply:
first use of eyes...it's not clear to me when use it maybe "I'll take a look"  :smile: